### PR TITLE
Links: bump x86_64 version

### PR DESCRIPTION
**Pull request by kenmays**
Closes https://github.com/haikuports/haikuports/issues/4467

### CODE REVIEW

File: www-client/links/links-2.20.2.recipe [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+COPYRIGHT="1999-2019 Mikulas Patocka
+	2000-2011 Petr Kulhavy, Karel Kulhavy, Martin Pergel"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://links.twibright.com/download/links-$portVersion.tar.bz2"
+CHECKSUM_SHA256="4b4f07d0e6261118d1365a5a5bfa31e1eafdbd280cfae6f0e9eedfea51a2f424"
+#PATCHES="links-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="!x86"
```
Comment (Begasus) [inline www-client/links/links-2.20.2.recipe:13]: This has been checked and not working for secondary architecture?

Comment (kenmays) [reply www-client/links/links-2.20.2.recipe:13]: Compiles fine on 2nd arch, but I mainly want to update the 64-bit one first.

Comment (Begasus) [reply www-client/links/links-2.20.2.recipe:13]: But why mark it as broken "!" then? You can have it for both archictectures then (32bit and 64bit)

Comment (kenmays) [reply www-client/links/links-2.20.2.recipe]: Fixed.

**Pull request merged**